### PR TITLE
Removes singloCanEat() from the repository

### PR DESCRIPTION
**Pull request by Probe1**
What's the worst that can happen?

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -531,7 +531,7 @@
 // SINGULOTH PULL REFACTOR
 /////////////////////////////
 /atom/movable/proc/canSingulothPull(var/obj/machinery/singularity/singulo)
-	return singuloCanEat()
+	return
```
Comment (gbasood) [inline code/game/atoms_movable.dm]: Isn't this going to change the default return value from 1 to 0?

Comment (clusterfack) [inline code/game/atoms_movable.dm]: MMM yeah that could be a problem

**Comment by Probe1** [comment]
tested against portals, the admin bus, docking ports.

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (Probe1) [inline code/game/atoms_movable.dm]: it was pullin plenty of shit in testing.

**Comment by gbasood** [comment]
``` DM
code/game/gamemodes/cult/narsie.dm:207:error: canPull: undefined proc
code/game/gamemodes/cult/narsie.dm:248:error: canPull: undefined proc
code/game/gamemodes/cult/narsie.dm:483:error: canPull: undefined proc
code/game/gamemodes/endgame/supermatter_cascade/portal.dm:61:error: canPull: undefined proc
code/game/gamemodes/cult/narsie.dm:291:error: canPull: undefined proc
```

**Comment by Probe1** [comment]
I wanted to tell you this was a bad idea but I kept my mouth shut.

Last commit removed.  You want shit deleted, ok I can help.  You want intelligent rewrites that rely on a fundamental understanding of byond coding, not so much.  Sorry.

**Comment by gbasood** [comment]
@Probe1 That said the return value I mentioned is still going to be important if we keep singuloCanPull

**Comment by Probe1** [comment]
Conflicts resolved.

**Pull request merged**